### DISCUSSION TITLE
Set command line options from a config file

### DISCUSSION THREAD

**Discussion by maburrub** [seleniumbase/SeleniumBase#1080] [Q&A] [answered]
I need to set the command line value from a config file instead of calling it from the command line for example instead of invoking the tests with --reruns=1 set it in a config file and read it from there.

**Comment by mdmintz** [comment, marked as answer]
@maburrub That's what a ``pytest.ini`` file is for. You can set default command-line arguments there.
See https://github.com/seleniumbase/SeleniumBase/blob/master/examples/pytest.ini for an example one.
